platforms/documentation/docs/src/docs/userguide/running-builds/tutorial/part1_gradle_init.adoc

```diff
@@ -84,15 +84,14 @@ NOTE: In this tutorial, all examples are macOS based.
 
 When you are done, the directory should look as follows:
 
-[.multi-language-sample]
 =====
-[source, kotlin]
+[source, text]
```

Comment:
❌ Please leave the `[.multi-language-sample]` part as well as the DSL flavor. This is an example for Kotlin DSL and below is the example for Groovy DSL - they are slightly different.

Reply:
But it's a directory listing, why is it even marked with a language? I've built the docs and it seem like removing the `[.mutli-language-sample]` only removes the "Kotlin / Groovy" tabs. Which is fine for this sample, because it is neither of them.

Are you sure it should be preserved?

Reply:
PTAL at file extensions: it's `gradle.kts` vs just `.gradle`

Reply:
Ok, I see. It seems like a hack TBH. A directory listing marked as Kotlin / Groovy.

Now fixed, though.